docs/UserGuide.md

```diff
@@ -11,18 +11,18 @@
    > Having any Java 8 version is not enough. <br>
    This app will not work with earlier versions of Java 8.
    
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
+1. Download the latest `lifekeeper.jar` from the [releases](../../../releases) tab.
+2. Copy the file to the folder you want to use as the home folder for your Lifekeeper.
 3. Double-click the file to start the app. The GUI should appear in a few seconds. 
-   > <img src="images/Ui.png" width="600">
+   > <img src="images/UIprototype.png" width="600">
```

Comment:
The right pane is a dashboard, the diagram refers to specific tasks which are saved.